src/components/ValidateSubmitComponent.js

```diff
@@ -0,0 +1,24 @@
+import { calculationTypeEnums } from "constants/enums";
+
+const isSubmitDisabled = (
```

Comment:
Move this under /utils or right under the UserInputForm component (this doesn't return a JSX component)

Reply:
in utils now